src/modules/expression/binop/add.rs

```diff
@@ -62,12 +62,13 @@ impl TranslateModule for Add {
     fn translate(&self, meta: &mut TranslateMetadata) -> String {
         let left = self.left.translate_eval(meta, false);
         let right = self.right.translate_eval(meta, false);
-        let quote = meta.gen_quote();
         match self.kind {
             Type::Array(_) => {
+                let quote = meta.gen_quote();
                 let id = meta.gen_array_id();
-                meta.stmt_queue.push_back(format!("__AMBER_ARRAY_ADD_{id}=({left} {right})"));
-                format!("{quote}${{__AMBER_ARRAY_ADD_{id}[@]}}{quote}")
+                let name = format!("__AMBER_ARRAY_ADD_{id}");
```

Comment:
We'll need to pass `name` into a new `TranslateModule` function in the  next PR, so it makes sense to store it in a local variable here.